frontend/src/components/CalendarView/CalendarView.jsx

```diff
@@ -128,18 +131,36 @@ function CalendarView () {
           allDay: info.event.allDay
         }
         const backendUrlAccess = import.meta.env.VITE_BACKEND_ADDRESS;
-        const options = {
-        method: 'PUT',
-        headers: {
-            'Accept': 'application/json',
-            'Content-Type': 'application/json'},
-        body: JSON.stringify(updatedEvent),
-        credentials: 'include'
-          };
-        const response = await fetch(`${backendUrlAccess}/calendar/events/${info.event.id}`, options);
-        if (!response.ok) {
-          throw new Error('Something went wrong!');
+
+        // Handles for Editing Google Calendar Events
```

Comment:
I think you can move all these google calendar cases to a separate file, it's bloating this calendar view file, which should primarily be for creating the UI